gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -292,7 +292,12 @@ gst_tensor_aggregator_init (GstTensorAggregator * self)
   self->frames_dim = DEFAULT_FRAMES_DIMENSION;
   self->concat = DEFAULT_CONCAT;
 
+  self->tensor_configured = FALSE;
+  gst_tensor_config_init (&self->in_config);
+  gst_tensor_config_init (&self->out_config);
+
   self->adapter = gst_adapter_new ();
+  g_assert (self->adapter != NULL);
```

Comment:
Is this necessary?

Reply:
yes, unnecessary :)

Reply:
Q) When `gst_adapter_new` fails to allocate new adapter, is it okay not to check here?

Reply:
then, fatal exception.

Reply:
Thanks! I removed the ineffective lines.